cpp/oneapi/dal/backend/primitives/selection/kselect_by_rows_impl.hpp

```diff
@@ -73,9 +75,6 @@ kselect_by_rows<Float>::kselect_by_rows(sycl::queue& queue,
 
 #define INSTANTIATE(F) template class kselect_by_rows<F>;
```

Comment:
I suggest removing macro, and instantiate `template class kselect_by_rows<F>` directly in cpps.